packages/@sfcc-bff/productapi/README.md

```diff
@@ -0,0 +1,9 @@
+# `@sfcc-bff/productapi`
+The GraphQL type definitions and resolvers for the Product API.
+
+## Usage
+Just by importing the package, the `productapi` will register itself with the @sfcc-core/core-graphql
```

Comment:
We may want to call out that all bff api extensions need to be imported before calling core-graphql.start(). start() is the actually mechanism which aggregates all the schema/resolvers before starting the graphql server.